### PR TITLE
Feature page

### PR DESCRIPTION
**Pull request by Haroonabdulrazaq**
This project is focused on building the design hierarchy as displayed in smashing magazine.
It has a lot of sections like the:
-  Main section
- Text section
- Link section
- Footer section

### CODE REVIEW

**Comment by Haroonabdulrazaq** [comment]
Note: Logo breaks when the network is slow

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+    </header>
+
+    <main>
+      <div class="main-section">
+        <div class="upper-text">
+          <h2 class="div-text">Don’t Miss These Latest Articles</h2>
+        </div>
+        <div class="parent-grid-div">
+          <!-- Medium grey Section -->
+          <div class="main-content-div medium-gray">
```
Comment (addod19) [inline index.html]: Kindly use``` <article> instead of <div class="main-content-div medium-gray">```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+              <span>
+                <i class="far fa-comment-alt"></i>
+                Leave a comment
+              </span>
+              <span>Smashing Podcat <sup>16</sup></span>
+            </div>
+          </div>
+
+          <!-- Dark grey Section -->
+          <div class="main-content-div dark-gray">
```
Comment (addod19) [inline index.html]: Kindly use ```<article> instead of <div class="main-content-div dark-gray">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, great work with this great project :+1: Let us improve our code and make it even better.Kindly address the comments below: :slightly_smiling_face: 

- ```Nav```, each of the nav items uses div for positioning
ccp
```Currently``` this is what we have
ccp1
- ```3 articles rows```
gcc
- There is a lot of repeating codes in our CSS file, let us apply the DRY rule. For more info kindly check this [article](https://speckyboy.com/good-bad-css-practices/)
- Kindly use float property for positioning some of the elements

Kindly implement these changes and submit another code review
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: index.html (hunk 2 above)
Comment (rammazzoti2000) [reply index.html]: Kindly implement the change required by the previous TSE:
`Kindly use <article> instead of <div class="main-content-div dark-gray"`

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello team :wave: awesome project, great job :rocket: :tada: 
You guys did a great work also trying to implement some of the changes required by the previous TSE like using float on some of the elements, the three article rows and wrapping the nav elements inside div tag for positioning.
There are still some issues left that are preventing me from approving this project
### :stop_sign: Requires Changes :construction: :construction_worker: 
- [ ] The change required by the previous TSE related to the repeating codes in the CSS file have not been implemented although your last commit says you did so, please kindly apply the DRY rule for repeating styling rules. For more info kindly check this [article](https://speckyboy.com/good-bad-css-practices/)
- [ ] You still have to implement the change required by the previous TSE  by using `<article>` instead of `<div class="main-content-div dark-gray">` around line 93 in html file
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Comment by Haroonabdulrazaq** [comment]
We have fixed all repeating properties which have the same value

**Review by sumancrest0001 — APPROVED** [review]
****STATUS: APPROVED :100: 
Congratulation TEAM :clap: :clap: :+1:**** 
You have done a great job to fix issues raised by the previous TSEs. It looks good to me and it's a green signal from my side. Now, You can merge the PR to master and move to the next project.

**Happy coding!!!**

**Comment by Haroonabdulrazaq** [comment]
Thanks for taking the time to review our code :+1

**Pull request merged**